apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-note.presenter.tsx

```diff
@@ -0,0 +1,15 @@
+import { SelfConnectionNote } from '../../../notes/hooks/use-loadable-self-notes-for-resource';
+
+export const SelfNotePresenter = ({ text, id, context }: SelfConnectionNote): JSX.Element => (
+    <div data-testid={id}>
+        {/* TODO Use property presenter helper after rebasing */}
```

Comment:
Don't forget about this.

Reply:
This could use the `SinglePropertyPresenter` here, though the notes may need more space.